hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestCommitInRatis.java

```diff
@@ -132,10 +133,13 @@ private void shutdown() {
     }
   }
 
-
-  @Test
-  public void test2WayCommitForRetryfailure() throws Exception {
+  @ParameterizedTest
+  @ValueSource(strings = {"ALL_COMMITTED", "MAJORITY_COMMITTED"})
```

Comment:
Use Ratis constants rather than hard-coding them?

Reply:
Let me check it.

Reply:
It's changed to use @EnumSource, but still need to specify ALL_COMMITTED and MAJORITY_COMMITTED for the enum class RaftProtos.ReplicationLevel has other elements not applicable to Ozone client watch.